.github/labels.json

```diff
@@ -0,0 +1,57 @@
+[
+	{ "name": "💩 template incomplete", "color": "#4E342E" },
+	{ "name": "💩 template removed", "color": "#4E342E" },
+
+	{ "name": "b¹ 🐞 code-splitting", "color": "#F44336" },
+	{ "name": "b² 🐞 scope-hoisting", "color": "#F44336" },
+	{ "name": "b³ 🐞 tree-shaking", "color": "#F44336" },
+
+	{ "name": "c¹ ⋅ discussion", "color": "#1976D2" },
+	{ "name": "c² ⋅ feedback wanted", "color": "#F9A825" },
+	{ "name": "c³ ⋅ PR welcome", "color": "#1B5E20" },
+	{ "name": "c⁴ ⋅ need more info", "color": "#6A1B9A" },
+	{ "name": "c⁵ ⋅ question", "color": "#C2185B" },
+	{ "name": "c⁶ ⋅ request for comments", "color": "#BBDEFB" },
+
+	{ "name": "p¹ ⋅ electron", "color": "#B2DFDB" },
+	{ "name": "p² ⋅ linux", "color": "#B2DFDB" },
+	{ "name": "p³ ⋅ mac", "color": "#B2DFDB" },
+	{ "name": "p⁴ ⋅ windows", "color": "#B2DFDB" },
+
+	{ "name": "pr¹ 🔧 chore", "color": "#D7CCC8" },
+	{ "name": "pr² 🔧 docs", "color": "#D7CCC8" },
+	{ "name": "pr³ 🔧 feature", "color": "#D7CCC8" },
+	{ "name": "pr⁴ 🔧 fix", "color": "#D7CCC8" },
+	{ "name": "pr⁵ 🔧 performance", "color": "#D7CCC8" },
+	{ "name": "pr⁶ 🔧 refactor", "color": "#D7CCC8" },
+	{ "name": "pr⁷ 🔧 style", "color": "#D7CCC8" },
+	{ "name": "pr⁸ 🔧 test", "color": "#D7CCC8" },
+
+	{ "name": "s¹ 🔥🔥🔥 critical", "color": "#E53935" },
+	{ "name": "s² 🔥🔥 important", "color": "#FB8C00" },
+	{ "name": "s³ 🔥 nice to have", "color": "#FDD835" },
+	{ "name": "s⁴ 💧 low", "color": "#039BE5" },
+	{ "name": "s⁵ 💧💧 inconvenient", "color": "#c0e0f7" },
+
+	{ "name": "t¹ 🐞 bug", "color": "#F44336" },
+	{ "name": "t² 📚 documentation", "color": "#FDD835" },
+	{ "name": "t³ ✨ enhancement", "color": "#03a9f4" },
+	{ "name": "t⁴ ✨ feature", "color": "#8bc34A" },
+	{ "name": "t⁵ ⋅ regression", "color": "#0052cc" },
+	{ "name": "t⁶ ⋅ todo", "color": "#311B92" },
+	{ "name": "t⁷ ⋅ waiting on upstream", "color": "#0D47A1" },
+
+	{ "name": "v¹ ⋅ alpha", "color": "#CDDC39" },
+	{ "name": "v² ⋅ beta", "color": "#FFEB3B" },
+	{ "name": "v³ ⋅ major", "color": "#FF9800" },
+	{ "name": "v⁴ ⋅ minor", "color": "#FFC107" },
+	{ "name": "v⁵ ⋅ next", "color": "#CDDC39" },
```

Comment:
How are there labels intended to be used?

Reply:
If this is about in what form we expect a PR to be released, the categories I see are patch-minor-major and experimental, which is like minor but means that related options start with `experimental` and the feature can have breaking changes in minor versions. At least for now I would like to keep avoiding beta and pre-releases and rather double down on testing or use experimental options if there are stability concerns.

Reply:
For projects that don't leverage tools like standard-release and commit-config-conventional, the tags are used for grouping PRs for a particular type of release. I also use them to tag issues that I want to tackle for the `next` or `beta` releases upcoming. Also assists when setting up project boards. It's not a must-have, rather just nice to have.

Reply:
I've typically omitted `patch` because anything not one of the others is a patch.

Reply:
Considering the current workflow of basically releasing patches as they are ready, I think it would be useful to boil this down to `major`, `minor`, `patch`, and maybe `minor (experimental)`.

Reply:
OK I'll change that up